lib/flame/pool.ex

```diff
@@ -522,7 +522,6 @@ defmodule FLAME.Pool do
       runner_count: runner_count(state),
       waiting_count: waiting_count(state),
       pending_count: pending_count(state),
-      desired_count: desired_count(state),
```

Comment:
with QueryNabooStrategy this introduces a sync block on every poll of the pool metrics